core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1540,4 +1540,12 @@ private SqlLibraryOperators() {
           OperandTypes.DATETIME,
           SqlFunctionCategory.TIMEDATE);
 
+
+  @LibraryOperator(libraries = {TERADATA, SNOWFLAKE})
+  public static final SqlFunction GETBIT =
+      new SqlFunction("GETBIT",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.INTEGER_NULLABLE, null,
+          OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.NUMERIC),
```

Comment:
Kindly change SqlTypeFamily to INTEGER, Since GETBIT doesn't accept float values.

Reply:
Done